### DISCUSSION TITLE
function to add a game to the DB

### DISCUSSION THREAD

**Discussion by Javiert54** [dyang886/Game-Save-Manager#16] [Ideas]
I would like the possibility to add games to the local data base, so if the game is not supported by game save manager, y can still backup the saves specifying mannually the game title and the path to the game save

**Comment by dyang886** [comment]
This application is currently undergoing a huge update with reimplemented functionalities, and it will certainly support custom save paths! Here's what it looks like in its current stage:
![Screenshot 2024-07-30 201405](https://github.com/user-attachments/assets/683757e5-c6f4-445b-a544-1960aa9a497b)
